kitsune/sumo/static/sumo/tpl/search-results.html

```diff
@@ -4,24 +4,6 @@
     {% if product %}
       {% set base_doctype_url = base_doctype_url|urlparams(product=product.slug) %}
     {% endif %}
-    <ul id="doctype-filter" class="search-filter sidebar-nav">
-      <li {{ w|class_selected(3) }}>
-        <a href="#" data-instant-search-set-params="w=3" data-instant-search="link" data-href="{{ base_doctype_url|encodeURI }}">
-          {{ _('Show Everything') }}
-        </a>
-      </li>
-      <li {{ w|class_selected(1) }}>
-        <a href="#" data-instant-search-set-params="w=1" data-instant-search="link" data-href="{{ base_doctype_url|urlparams(w=1)|encodeURI }}">
-          {{ _('Help Articles Only') }}
-        </a>
-      </li>
-      <li {{ w|class_selected(2) }}>
-        <a href="#" data-instant-search-set-params="w=2" data-instant-search="link" data-href="{{ base_doctype_url|urlparams(w=2)|encodeURI }}">
-          {{ _('Community Discussion Only') }}
-        </a>
-      </li>
-    </ul>
-
     {% set base_product_url = base_url %}
     {% if w != 3 %}
       {% set base_product_url = base_product_url|urlparams(w=w) %}
```

Comment:
I'm not seeing any UI response when I click on one of the products in the sidebar list when searching via the sticky nav. I see a network request is made, but there's no visual indication that one of the selections (e.g. "All Products", "Firefox") has been chosen/applied as a filter.

Reply:
The product gets highlighted when you click on it and it shows a black left border meaning it's selected though. I know before the search title wasn't all bold and it showed the product selected in bold in the title. Now it's in the title but we can't really tell what is the product unless you look at the selected product in the sidear. Maybe show the product as a link in the title?

![image](https://user-images.githubusercontent.com/216256/42395223-8040b82e-8133-11e8-97ac-784dfee66640.png)

Reply:
Are you getting the UI from the above screenshot using the instant/live search? I don't get any bolding - from this PR locally or even in prod. It seems the `selected` class attribute is being rendered incorrectly:

![screen shot 2018-07-06 at 3 26 03 pm](https://user-images.githubusercontent.com/317532/42398799-f227070c-8130-11e8-96bd-e4e985ceded8.png)

As this appears to be a pre-existing bug, I suppose we don't need to block this PR. I'll file a separate issue.

Reply:
A quick test shows that changing `return ' class="selected" ';` to `return ' class=selected ';` in [`/kitsune/sumo/static/js/nunjucks.js#53`](https://github.com/mozilla/kitsune/blob/master/kitsune/sumo/static/sumo/js/nunjucks.js#L53) solves the issue.

If you want to double-check that and include it here, please feel free. Otherwise, will file an issue and separate PR.

Thanks!

Reply:
Yeah I got that screenshot from the instant/live search. I don't see the second set of double quotes like you do. I see:

![image](https://user-images.githubusercontent.com/216256/42416683-3ff6d2ca-824c-11e8-8456-e77e391dba36.png)